contract/cache/solidity-files-cache.json

```diff
@@ -1,9 +1,15 @@
 {
   "_format": "hh-sol-cache-2",
   "files": {
+<<<<<<< HEAD
```

Comment:
Seems like there was an unresolved conflict here, best to just remove this cache file.